src/create-glamorous.js

```diff
@@ -78,7 +78,10 @@ export default function createGlamorous(splitProps) {
         }
 
         render() {
-          const props = this.props
+          const {children, ...props} = this.props
+          const spreadableChildren = Array.isArray(children) ?
```

Comment:
Is there a `React.Children` helper that could clean this up at all? Maybe [`React.Children.toArray`](https://facebook.github.io/react/docs/react-api.html#react.children.toarray)?

Reply:
That's a good point, and with your question below, it makes me realize we shouldn't be spreading the `children` without using `React.Children.toArray` since it's an opaque data structure.